db/src/debug.rs

```diff
@@ -21,13 +21,20 @@ use rusqlite::types::{ToSql};
 use tabwriter::TabWriter;
 
 use bootstrap;
+use db::TypedSQLValue;
```

Comment:
I haven't been separating out `mentat_*` from `samecratemodule_*`.  I can start doing that.